versions.props

```diff
@@ -8,7 +8,7 @@ com.google.errorprone:error_prone_core = 2.3.1
 com.google.errorprone:error_prone_test_helpers = 2.3.1
 com.google.guava:* = 18.0
 com.palantir.baseline:* = 0.29.4
-com.palantir.remoting3:* = 3.5.1
+com.palantir.tracing:* = 1.1.0
```

Comment:
@qinfchen Do we have a gradle substitute rule for this for downstream consumers?

Reply:
actually, nevermind as we're changing packages as well so consumers can have both remoting3 & new tracing coexist peacefully, though we'll want all tracing using a single `Tracer`

Reply:
having both co-existing is actually quite concerning though, because if  `com.palantir.remoting3.Tracer#getTraceId()` and `com.palantir.tracing.Tracer#getTraceId()` disagree then you'd be in a pretty weird state.  Also the logging subscribers will probably only be attached to one of them!